css/src/components/persona.scss

```diff
@@ -0,0 +1,57 @@
+$persona-font-size-sm: $font-size-9 !default;
```

Comment:
Funny the diff here is showing additional changes. This should be in `main` already, remove and rebase?